spec/commands/v2/put_content_spec.rb

```diff
@@ -812,5 +813,77 @@
 
       described_class.call(payload)
     end
+
+    context "draft does not exist" do
+      context "with a provided last_edited_at" do
+        it "stores the provided timestamp" do
+          last_edited_at = 1.year.ago
+
+          described_class.call(payload.merge(
+                                 last_edited_at: last_edited_at
+                              ))
+
+          content_item = ContentItem.last
+
+          expect(content_item.last_edited_at.iso8601).to eq(last_edited_at.iso8601)
+        end
+      end
+    end
+
+    context "draft does exist" do
+      let!(:content_item) {
+        FactoryGirl.create(:draft_content_item,
+          content_id: content_id,
+        )
+      }
+
+      context "with a provided last_edited_at" do
+        it "stores the provided timestamp" do
+          last_edited_at = 1.year.ago
+
+          described_class.call(payload.merge(
+                                 last_edited_at: last_edited_at
+                               ))
+
+          content_item.reload
+
+          expect(content_item.last_edited_at.iso8601).to eq(last_edited_at.iso8601)
+        end
+      end
+    end
+
+    context "with no provided last_edited_at" do
+      let!(:content_item) {
+        FactoryGirl.create(:draft_content_item,
+          content_id: content_id,
+        )
+      }
+
+      context "when update type is major or minor" do
+        it "stores last_edited_at as the current time" do
+          Timecop.freeze do
+            described_class.call(payload)
+
+            content_item.reload
+
+            expect(content_item.last_edited_at.iso8601).to eq(Time.zone.now.iso8601)
+          end
+        end
+      end
+
+      context "when other update type" do
+        it "dosen't change last_edited_at" do
+          old_last_edited_at = content_item.last_edited_at
+
+          described_class.call(payload.merge(
+                                 update_type: "republish"
+                               ))
+
+          content_item.reload
+
+          expect(content_item.last_edited_at).to eq(old_last_edited_at)
+        end
+      end
+    end
```

Comment:
Yeah I think we should distinguish between the client sending `nil` and not sending the param at all.

If it sends `nil` we shouldn't do anything because it's explicitly neither major nor minor, but if they don't send anything we should assume major or minor because the most common use case for that will be apps which choose to send the update type on publish, and thus part of the normal editor workflow (which only deals in minor or major updates).

Does that sound right?